nautobot/core/views/renderers.py

```diff
@@ -65,10 +65,14 @@ def construct_table(self, view, **kwargs):
         table_class = view.get_table_class()
         request = kwargs.get("request", view.request)
         queryset = view.alter_queryset(request)
+
         if view.action in ["list", "notes", "changelog"]:
             if view.action == "list":
                 permissions = kwargs.get("permissions", {})
-                table = table_class(queryset, user=request.user)
+                order_by = view.request.GET.getlist("sort")
```

Comment:
I note that the `notes` and `changelog` cases below aren't taking `order_by`/`sort` into account. Is that another bug, or are those tables non-orderable?

Reply:
It looks like this is redundant because it's happening in the `RequestConfig.configure` [here](https://github.com/jieter/django-tables2/blob/c592b7b9fcff956499507c38d0f80b7e8b76666e/django_tables2/config.py#L42-L44)? (`table.prefixed_order_by_field` defaults to `"sort"`)